op-node/p2p/gossip.go

```diff
@@ -424,11 +437,18 @@ func JoinGossip(p2pCtx context.Context, self peer.ID, ps *pubsub.PubSub, log log
 	}
 	go LogTopicEvents(p2pCtx, log.New("topic", "blocks"), blocksTopicEvents)
 
-	// TODO: block topic scoring parameters
+	// TODO: make this configurable behind a cli flag - disabled or default
+	// Set default block topic scoring parameters
```

Comment:
I'd recommend to name flags based on what they do, rather than on what's default. Then the CLI flag definition is the only place that changes the default behavior.